.github/workflows/test.yml

```
@@ -27,16 +28,163 @@ jobs:
+            
+            // Calculate averages
+            const avgStatements = (totalStatements / packageCount).toFixed(1);
+            const avgBranches = (totalBranches / packageCount).toFixed(1);
+            const avgFunctions = (totalFunctions / packageCount).toFixed(1);
+            const avgLines = (totalLines / packageCount).toFixed(1);
+            
+            const getEmoji = (pct) => pct >= 80 ? '🟢' : pct >= 60 ? '🟡' : '🔴';
+            
+            const body = `## 📊 Test Coverage Report
+            
+| Package | Statements | Branches | Functions | Lines |
```

actionlint (1.7.10)

could not parse as YAML: did not find expected comment or line break

(syntax-check)